meshroom/ui/qml/Viewer/Viewer2D.qml

```diff
@@ -1096,6 +1096,9 @@ FocusScope {
                         }
                         MaterialToolButton {
                             id: displayLensDistortionViewer
+
+                            property int number_changes: null
+                            property bool previous_checked: null
```

Comment:
Variable names should ideally respect the case (here, camelCase) that's used across the file.

```suggestion
                            property int numberChanges: null
                            property bool previousChecked: null
```